src/sparql/doid-relevant-signature.sparql

```diff
@@ -2,9 +2,15 @@
 SELECT DISTINCT ?term
 WHERE {
   { 
-    { ?s1 ?p1 ?term . }
+    { 
+     ?s1 ?p1 ?term . 
+     ?term rdfs:subClassOf* <http://purl.obolibrary.org/obo/DOID_4> .
```

Comment:
ah, i see, subClassOf. Yes, I was thinking that it would make sense to reference the root class here like this.